src/js/components/MaskedInput/doc.js

```diff
@@ -84,6 +84,12 @@ export const themeDoc = {
     type: 'string',
     defaultValue: '18px',
   },
+  'maskedInput.disabled.opacity': {
+    description: 'The opacity when the MaskedInput is disabled.',
+    type: 'number',
+    defaultValue: 'global.control.disabled.opacity',
```

Comment:
According to base.js, the `defaultValue` is `undefined`, and I think that is the value we should be explicit about.
```
...themeDocUtils.disabledStyle,
```
will be documenting the `global.control.disabled.opacity` property independently.